### PR TITLE
obfuscated code post

### PR DESCRIPTION
**Pull request by hghimanshu**

### CODE REVIEW

File: source/_posts/Creating-obfuscated-code-in-python.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+
+---
+title: Creating obfuscated code in python
```
Comment (Rajeev111974) [inline source/_posts/Creating-obfuscated-code-in-python.md:3]: Title   tag character should be 50-60 character

File: source/_posts/Creating-obfuscated-code-in-python.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+
```
Comment (Rajeev111974) [inline source/_posts/Creating-obfuscated-code-in-python.md:1]: Category should be mentioned. it is not here right now

**Comment by Rajeev111974** [comment]
HI Himanshu,

 

Please check the blog as we have mentioned some changes there which need to update

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

File: source/_posts/Creating-obfuscated-code-in-python.md (hunk 1 above)
Comment (hghimanshu) [reply source/_posts/Creating-obfuscated-code-in-python.md]: done, kindly check

File: source/_posts/Creating-obfuscated-code-in-python-to-make-them-unreadable.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+
+---
+title: Creating obfuscated code in python to make them unreadable 
+date: 2020-08-23 10:00:00
+author: Himanshu Garg
+categories:
+  - ["Scripting"]
+tags:
```
Comment (Rajeev111974) [inline source/_posts/Creating-obfuscated-code-in-python-to-make-them-unreadable.md:8]: put 3 and 4 keyword from given keywords list and use in content also

NAYAN
Traffic monitoring
AI
Livelihood
Driver analytics
Violation mapping
Better commute
Monitor infrastructure defects
Safer roads
Road Safety
Traffic Violation
Artificial Intelligence
India
Smart City
Safety
Data Protection
PARKING DETECTION,
VEHICLE OCCUPANCY DETECTION
NAYANINDIA
USER INTERACTION
NAYAN IP
NAYAN DATA PROTECTION
Road User
Public Road
Vehicle
Dangers
MACHINE LEARNING

Comment (hghimanshu) [reply source/_posts/Creating-obfuscated-code-in-python-to-make-them-unreadable.md:8]: done, please check

**Comment by Rajeev111974** [comment]
Dear Abhishek,

 

Blog reviewed and it is ready for live, Please do the thing needful.

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

**Review by Rajeev111974 — APPROVED** [review]

**Review by rana-abhishek — APPROVED** [review]

**Pull request merged**